Binary files openssl-0.9.8za/src/doc/.DS_Store and ./doc/.DS_Store differ
diff -Nur openssl-0.9.8za/src/doc/apps/smime.pod ./doc/apps/smime.pod
--- openssl-0.9.8za/src/doc/apps/smime.pod	2014-06-05 01:09:08.000000000 -0700
+++ ./doc/apps/smime.pod	2014-08-05 17:21:40.000000000 -0700
@@ -265,28 +265,28 @@
 
 =over 4
 
-=item 0
+=item B<0>
 
 the operation was completely successfully.
 
-=item 1 
+=item B<1> 
 
 an error occurred parsing the command options.
 
-=item 2
+=item B<2>
 
 one of the input files could not be read.
 
-=item 3
+=item B<3>
 
 an error occurred creating the PKCS#7 file or when reading the MIME
 message.
 
-=item 4
+=item B<4>
 
 an error occurred decrypting or verifying the message.
 
-=item 5
+=item B<5>
 
 the message was verified correctly but an error occurred writing out
 the signers certificates.
diff -Nur openssl-0.9.8za/src/doc/crypto/rand.pod ./doc/crypto/rand.pod
--- openssl-0.9.8za/src/doc/crypto/rand.pod	2014-06-05 00:22:48.000000000 -0700
+++ ./doc/crypto/rand.pod	2014-08-05 17:21:40.000000000 -0700
@@ -74,16 +74,16 @@
 
 =over 4
 
-=item 1
+=item B<1>
 
 A good hashing algorithm to mix things up and to convert the RNG 'state'
 to random numbers.
 
-=item 2
+=item B<2>
 
 An initial source of random 'state'.
 
-=item 3
+=item B<3>
 
 The state should be very large.  If the RNG is being used to generate
 4096 bit RSA keys, 2 2048 bit random strings are required (at a minimum).
@@ -93,13 +93,13 @@
 a bad idea to keep quite a lot of RNG state.  It should be easier to
 break a cipher than guess the RNG seed data.
 
-=item 4
+=item B<4>
 
 Any RNG seed data should influence all subsequent random numbers
 generated.  This implies that any random seed data entered will have
 an influence on all subsequent random numbers generated.
 
-=item 5
+=item B<5>
 
 When using data to seed the RNG state, the data used should not be
 extractable from the RNG state.  I believe this should be a
@@ -108,12 +108,12 @@
 not be disclosed by either subsequent random numbers or a
 'core' dump left by a program crash.
 
-=item 6
+=item B<6>
 
 Given the same initial 'state', 2 systems should deviate in their RNG state
 (and hence the random numbers generated) over time if at all possible.
 
-=item 7
+=item B<7>
 
 Given the random number output stream, it should not be possible to determine
 the RNG state or the next random number.
diff -Nur openssl-0.9.8za/src/doc/ssl/SSL_COMP_add_compression_method.pod ./doc/ssl/SSL_COMP_add_compression_method.pod
--- openssl-0.9.8za/src/doc/ssl/SSL_COMP_add_compression_method.pod	2014-06-05 01:09:08.000000000 -0700
+++ ./doc/ssl/SSL_COMP_add_compression_method.pod	2014-08-05 17:21:40.000000000 -0700
@@ -53,11 +53,11 @@
 
 =over 4
 
-=item 0
+=item B<0>
 
 The operation succeeded.
 
-=item 1
+=item B<1>
 
 The operation failed. Check the error queue to find out the reason.
 
diff -Nur openssl-0.9.8za/src/doc/ssl/SSL_CTX_add_session.pod ./doc/ssl/SSL_CTX_add_session.pod
--- openssl-0.9.8za/src/doc/ssl/SSL_CTX_add_session.pod	2014-06-05 01:09:08.000000000 -0700
+++ ./doc/ssl/SSL_CTX_add_session.pod	2014-08-05 17:21:40.000000000 -0700
@@ -52,13 +52,13 @@
 
 =over 4
 
-=item 0
+=item B<0>
 
  The operation failed. In case of the add operation, it was tried to add
  the same (identical) session twice. In case of the remove operation, the
  session was not found in the cache.
 
-=item 1
+=item B<1>
  
  The operation succeeded.
 
diff -Nur openssl-0.9.8za/src/doc/ssl/SSL_CTX_load_verify_locations.pod ./doc/ssl/SSL_CTX_load_verify_locations.pod
--- openssl-0.9.8za/src/doc/ssl/SSL_CTX_load_verify_locations.pod	2014-06-05 01:09:08.000000000 -0700
+++ ./doc/ssl/SSL_CTX_load_verify_locations.pod	2014-08-05 17:21:40.000000000 -0700
@@ -100,13 +100,13 @@
 
 =over 4
 
-=item 0
+=item B<0>
 
 The operation failed because B<CAfile> and B<CApath> are NULL or the
 processing at one of the locations specified failed. Check the error
 stack to find out the reason.
 
-=item 1
+=item B<1>
 
 The operation succeeded.
 
diff -Nur openssl-0.9.8za/src/doc/ssl/SSL_CTX_set_client_CA_list.pod ./doc/ssl/SSL_CTX_set_client_CA_list.pod
--- openssl-0.9.8za/src/doc/ssl/SSL_CTX_set_client_CA_list.pod	2014-06-05 01:09:08.000000000 -0700
+++ ./doc/ssl/SSL_CTX_set_client_CA_list.pod	2014-08-05 17:21:40.000000000 -0700
@@ -66,13 +66,13 @@
 
 =over 4
 
-=item 0
+=item B<0>
 
 A failure while manipulating the STACK_OF(X509_NAME) object occurred or
 the X509_NAME could not be extracted from B<cacert>. Check the error stack
 to find out the reason.
 
-=item 1
+=item B<1>
 
 The operation succeeded.
 
diff -Nur openssl-0.9.8za/src/doc/ssl/SSL_CTX_set_session_id_context.pod ./doc/ssl/SSL_CTX_set_session_id_context.pod
--- openssl-0.9.8za/src/doc/ssl/SSL_CTX_set_session_id_context.pod	2014-06-05 01:09:08.000000000 -0700
+++ ./doc/ssl/SSL_CTX_set_session_id_context.pod	2014-08-05 17:21:40.000000000 -0700
@@ -64,13 +64,13 @@
 
 =over 4
 
-=item 0
+=item B<0>
 
 The length B<sid_ctx_len> of the session id context B<sid_ctx> exceeded
 the maximum allowed length of B<SSL_MAX_SSL_SESSION_ID_LENGTH>. The error
 is logged to the error stack.
 
-=item 1
+=item B<1>
 
 The operation succeeded.
 
diff -Nur openssl-0.9.8za/src/doc/ssl/SSL_CTX_set_ssl_version.pod ./doc/ssl/SSL_CTX_set_ssl_version.pod
--- openssl-0.9.8za/src/doc/ssl/SSL_CTX_set_ssl_version.pod	2014-06-05 01:09:08.000000000 -0700
+++ ./doc/ssl/SSL_CTX_set_ssl_version.pod	2014-08-05 17:21:40.000000000 -0700
@@ -42,11 +42,11 @@
 
 =over 4
 
-=item 0
+=item B<0>
 
 The new choice failed, check the error stack to find out the reason.
 
-=item 1
+=item B<1>
 
 The operation succeeded.
 
diff -Nur openssl-0.9.8za/src/doc/ssl/SSL_accept.pod ./doc/ssl/SSL_accept.pod
--- openssl-0.9.8za/src/doc/ssl/SSL_accept.pod	2014-06-05 01:09:08.000000000 -0700
+++ ./doc/ssl/SSL_accept.pod	2014-08-05 17:21:40.000000000 -0700
@@ -44,13 +44,13 @@
 
 =over 4
 
-=item 0
+=item B<0>
 
 The TLS/SSL handshake was not successful but was shut down controlled and
 by the specifications of the TLS/SSL protocol. Call SSL_get_error() with the
 return value B<ret> to find out the reason.
 
-=item 1
+=item B<1>
 
 The TLS/SSL handshake was successfully completed, a TLS/SSL connection has been
 established.
diff -Nur openssl-0.9.8za/src/doc/ssl/SSL_clear.pod ./doc/ssl/SSL_clear.pod
--- openssl-0.9.8za/src/doc/ssl/SSL_clear.pod	2014-06-05 01:09:08.000000000 -0700
+++ ./doc/ssl/SSL_clear.pod	2014-08-05 17:21:40.000000000 -0700
@@ -56,12 +56,12 @@
 
 =over 4
 
-=item 0
+=item B<0>
 
 The SSL_clear() operation could not be performed. Check the error stack to
 find out the reason.
 
-=item 1
+=item B<1>
 
 The SSL_clear() operation was successful.
 
diff -Nur openssl-0.9.8za/src/doc/ssl/SSL_connect.pod ./doc/ssl/SSL_connect.pod
--- openssl-0.9.8za/src/doc/ssl/SSL_connect.pod	2014-06-05 01:09:08.000000000 -0700
+++ ./doc/ssl/SSL_connect.pod	2014-08-05 17:21:40.000000000 -0700
@@ -41,13 +41,13 @@
 
 =over 4
 
-=item 0
+=item B<0>
 
 The TLS/SSL handshake was not successful but was shut down controlled and
 by the specifications of the TLS/SSL protocol. Call SSL_get_error() with the
 return value B<ret> to find out the reason.
 
-=item 1
+=item B<1>
 
 The TLS/SSL handshake was successfully completed, a TLS/SSL connection has been
 established.
diff -Nur openssl-0.9.8za/src/doc/ssl/SSL_do_handshake.pod ./doc/ssl/SSL_do_handshake.pod
--- openssl-0.9.8za/src/doc/ssl/SSL_do_handshake.pod	2014-06-05 01:09:08.000000000 -0700
+++ ./doc/ssl/SSL_do_handshake.pod	2014-08-05 17:21:40.000000000 -0700
@@ -45,13 +45,13 @@
 
 =over 4
 
-=item 0
+=item B<0>
 
 The TLS/SSL handshake was not successful but was shut down controlled and
 by the specifications of the TLS/SSL protocol. Call SSL_get_error() with the
 return value B<ret> to find out the reason.
 
-=item 1
+=item B<1>
 
 The TLS/SSL handshake was successfully completed, a TLS/SSL connection has been
 established.
diff -Nur openssl-0.9.8za/src/doc/ssl/SSL_read.pod ./doc/ssl/SSL_read.pod
--- openssl-0.9.8za/src/doc/ssl/SSL_read.pod	2014-06-05 01:09:08.000000000 -0700
+++ ./doc/ssl/SSL_read.pod	2014-08-05 17:21:40.000000000 -0700
@@ -86,7 +86,7 @@
 The read operation was successful; the return value is the number of
 bytes actually read from the TLS/SSL connection.
 
-=item 0
+=item B<0>
 
 The read operation was not successful. The reason may either be a clean
 shutdown due to a "close notify" alert sent by the peer (in which case
diff -Nur openssl-0.9.8za/src/doc/ssl/SSL_session_reused.pod ./doc/ssl/SSL_session_reused.pod
--- openssl-0.9.8za/src/doc/ssl/SSL_session_reused.pod	2014-06-05 01:09:08.000000000 -0700
+++ ./doc/ssl/SSL_session_reused.pod	2014-08-05 17:21:40.000000000 -0700
@@ -27,11 +27,11 @@
 
 =over 4
 
-=item 0
+=item B<0>
 
 A new session was negotiated.
 
-=item 1
+=item B<1>
 
 A session was reused.
 
diff -Nur openssl-0.9.8za/src/doc/ssl/SSL_set_fd.pod ./doc/ssl/SSL_set_fd.pod
--- openssl-0.9.8za/src/doc/ssl/SSL_set_fd.pod	2014-06-05 01:09:08.000000000 -0700
+++ ./doc/ssl/SSL_set_fd.pod	2014-08-05 17:21:40.000000000 -0700
@@ -35,11 +35,11 @@
 
 =over 4
 
-=item 0
+=item B<0>
 
 The operation failed. Check the error stack to find out why.
 
-=item 1
+=item B<1>
 
 The operation succeeded.
 
diff -Nur openssl-0.9.8za/src/doc/ssl/SSL_set_session.pod ./doc/ssl/SSL_set_session.pod
--- openssl-0.9.8za/src/doc/ssl/SSL_set_session.pod	2014-06-05 01:09:08.000000000 -0700
+++ ./doc/ssl/SSL_set_session.pod	2014-08-05 17:21:40.000000000 -0700
@@ -37,11 +37,11 @@
 
 =over 4
 
-=item 0
+=item B<0>
 
 The operation failed; check the error stack to find out the reason.
 
-=item 1
+=item B<1>
 
 The operation succeeded.
 
diff -Nur openssl-0.9.8za/src/doc/ssl/SSL_set_shutdown.pod ./doc/ssl/SSL_set_shutdown.pod
--- openssl-0.9.8za/src/doc/ssl/SSL_set_shutdown.pod	2014-06-05 01:09:08.000000000 -0700
+++ ./doc/ssl/SSL_set_shutdown.pod	2014-08-05 17:21:40.000000000 -0700
@@ -24,7 +24,7 @@
 
 =over 4
 
-=item 0
+=item B<0>
 
 No shutdown setting, yet.
 
diff -Nur openssl-0.9.8za/src/doc/ssl/SSL_shutdown.pod ./doc/ssl/SSL_shutdown.pod
--- openssl-0.9.8za/src/doc/ssl/SSL_shutdown.pod	2014-06-05 01:09:08.000000000 -0700
+++ ./doc/ssl/SSL_shutdown.pod	2014-08-05 17:21:40.000000000 -0700
@@ -92,14 +92,14 @@
 
 =over 4
 
-=item 0
+=item B<0>
 
 The shutdown is not yet finished. Call SSL_shutdown() for a second time,
 if a bidirectional shutdown shall be performed.
 The output of L<SSL_get_error(3)|SSL_get_error(3)> may be misleading, as an
 erroneous SSL_ERROR_SYSCALL may be flagged even though no error occurred.
 
-=item 1
+=item B<1>
 
 The shutdown was successfully completed. The "close notify" alert was sent
 and the peer's "close notify" alert was received.
diff -Nur openssl-0.9.8za/src/doc/ssl/SSL_write.pod ./doc/ssl/SSL_write.pod
--- openssl-0.9.8za/src/doc/ssl/SSL_write.pod	2014-06-05 01:09:08.000000000 -0700
+++ ./doc/ssl/SSL_write.pod	2014-08-05 17:21:40.000000000 -0700
@@ -79,7 +79,7 @@
 The write operation was successful, the return value is the number of
 bytes actually written to the TLS/SSL connection.
 
-=item 0
+=item B<0>
 
 The write operation was not successful. Probably the underlying connection
 was closed. Call SSL_get_error() with the return value B<ret> to find out,
